tests/schema/fixtures/dashboard-0005.json

```diff
@@ -13,14 +13,15 @@
                             "error": false,
                             "fill": 1,
                             "lines": true,
-                            "linewidth": 2,
+                            "linewidth": 1,
                             "percentage": false,
                             "pointradius": 5,
                             "points": false,
                             "span": 12,
                             "stack": false,
                             "steppedLine": false,
                             "targets": [],
+                            "thresholds": [],
```

Comment:
I think a better way to handle this situation is to make thresholds field optional instead of mandatory. This line needs to be changed: https://github.com/deliveryhero/grafyaml/blob/master/grafana_dashboards/schema/panel/graph.py#L125
Then we wouldn't need to add this in every fixture

Reply:
I've done that but it doesn't negate the need to add `"thresholds": [],` to all the test templates?